src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral3.java

```diff
@@ -25,7 +25,7 @@ <T> void xx(List<T> m){}
     }
 
     public void method2() {
-        Integer a = 5;
+        Integer a = 5; // violation 'Distance .* is 2.'
```

Comment:
distance should be 1, as `class AClass extends Parent {` is class start not a scope, variable cannot be moved to any better place.
So no violation should be here